client/components/telecom/telephony/group/number/feature/ovhPabx/sound/popover-section/telephony-group-number-feature-ovh-pabx-sound-popover-section.html

```diff
@@ -1,42 +1,42 @@
 <section>
     <div class="section-group">
-        <label class="section-item btn-item full-width pointer input-file"
+        <label class="section-item btn-item w-100 input-file"
                data-telephony-number-ovh-pabx-sound-uploader
                data-ovh-pabx="$ctrl.ovhPabx"
                data-ng-model="$ctrl.model.soundFile"
                data-upload-errors="$ctrl.uploadErrors"
                data-ng-class="{ 'text-danger': $ctrl.model.soundFile && $ctrl.hasSoundFileError() }">
-            <div class="dtable">
-                <div class="dtd vertical-middle left-space-p16 right-space-p16">
+            <div class="d-table">
+                <div class="d-table-cell align-middle px-3">
                     <i class="ovh-font ovh-font-upload fs18"></i>
                 </div>
-                <div class="dtd vertical-middle"
+                <div class="d-table-cell align-middle"
                      data-ng-if="!$ctrl.hasSoundFileError()">
                     <span class="block radio-label-main"
                           data-translate="telephony_number_feature_ovh_pabx_sound_popover_section_add">
                     </span>
-                    <span class="small normal"
+                    <span class="small font-weight-normal"
                           data-translate="telephony_number_feature_ovh_pabx_sound_popover_section_add_explain">
                     </span>
                 </div>
-                <div class="dtd vertical-middle"
+                <div class="d-table-cell align-middle"
                      data-ng-if="$ctrl.model.soundFile && $ctrl.hasSoundFileError()">
                     <span class="block"
                           data-ng-bind="$ctrl.model.soundFile.name">
                     </span>
-                    <span class="small normal"
+                    <span class="small font-weight-normal"
```

Comment:
once again, it's an help block :)

Please check https://getbootstrap.com/docs/3.3/css/#forms-help-text

Reply:
It looks more of a list of warnings, not contextual help before data entry.